client/packages/invoices/src/Returns/api/api.ts

```diff
@@ -0,0 +1,14 @@
+import { Sdk } from './operations.generated';
```

Comment:
I don't understand the structure of these Api folders at all? Is it the same pattern you used for HSH?

I could really do with a lesson on how it works/why it's structured like this, as I find it a massive headache to do even the simplest things.

Reply:
Yeah... it sounds like it's a headache for everyone - we initially copied this pattern in HSH/Notify but refactored it out because its a lot of complexity without seeming to add much value...

Just kept it here to be consistent with the rest of the app, but I'd definitely be up for a refactor! Even one R&D day 🤔